scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/remotes/postgres/UpdateSchedulingGoalParameterSchemaAction.java

```diff
@@ -0,0 +1,38 @@
+package gov.nasa.jpl.aerie.scheduler.server.remotes.postgres;
+
+import gov.nasa.jpl.aerie.merlin.driver.json.ValueSchemaJsonParser;
+import gov.nasa.jpl.aerie.merlin.protocol.types.ValueSchema;
+import gov.nasa.jpl.aerie.scheduler.server.models.GoalId;
+import gov.nasa.jpl.aerie.scheduler.server.models.GoalRecord;
+import gov.nasa.jpl.aerie.scheduler.server.models.GoalType;
```

Comment:
These, `Path`, and `Optional` are all unused imports

Reply:
Done